hazelcast/src/main/java/com/hazelcast/spi/impl/operationexecutor/impl/OperationExecutorImpl.java

```diff
@@ -97,7 +99,7 @@ public final class OperationExecutorImpl implements OperationExecutor, StaticMet
     private final ILogger logger;
 
     // all operations for specific partitions will be executed on these threads, e.g. map.put(key, value)
-    private final PartitionOperationThreadImpl[] partitionThreads;
+    private final PartitionOperationThread[] partitionThreads;
```

Comment:
Reverting to the original code. PartitionOperationThreadImpl was introduced in the TPC branch, but there is no need or that any longer with the new approach.